client/cli/cli.go

```diff
@@ -63,27 +63,52 @@ func Run(run bool, pid int, args []string) {
 	}()
 
 	cmds := command.DebugCommands()
+	names := cmds.Names()
+
+	// set command completer
+	t.line.SetWordCompleter(func(line string, pos int) (head string, completions []string, tail string) {
+		start := strings.LastIndex(line[:pos], " ")
```

Comment:
This is to support various type of completion:
- if beginning of line you are looking for the command name
- if somewhere in the middle you are trying to do completion on some arguments. Right now it's just for "help" (complete using the command names) but for "info" as today it would complete on the subcommands, or for "break" it could complete on function names and "print" could complete on variable names (of course the completer code will have to be rearranged)